### PR TITLE
sysvar: add v6.5 sysvar

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12239
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1720,6 +1720,16 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 - This variables specifies whether to use the pipeline execution algorithm for window functions.
 
+### tidb_enable_plan_replayer_capture
+
+> Warning:
+>
+> The feature controlled by this variable is not fully functional in the current TiDB version. Do not change the default value.
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
```
Comment (TomShawn) [inline system-variables.md:1730]:
```suggestion
- Persists to cluster: Yes
- Type: Boolean
```

File: system-variables.md [hunk 2]
```diff
@@ -1720,6 +1720,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+- Type: Boolean
+- Default value: `FALSE`
```
Comment (TomShawn) [inline system-variables.md:1732]:
```suggestion
- Default value: `OFF`
```

File: system-variables.md [hunk 3]
```diff
@@ -1720,6 +1720,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Default value: `ON`
 - This variables specifies whether to use the pipeline execution algorithm for window functions.
 
+### tidb_enable_plan_replayer_capture
+
+> Warning:
+>
+> The feature controlled by this variable is not fully functional in the current TiDB version. Do not change the default value.
```
Comment (Oreoxmt) [inline system-variables.md:1727]:
```suggestion
> This variable controls a feature that is not fully functional in the current TiDB version. Do not change the default value.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**